telegram-bot/src/main/kotlin/com/ajaxproject/telegrambot/bot/handlers/impl/financehandlers/FinanceManageButtonsHandler.kt

```diff
@@ -30,12 +31,21 @@ class FinanceManageButtonsHandler(
         val localizationText = textService.textMap[dispatchRequest.updateSession.localization]
         return telegramService.sendMessage(
             chatId = dispatchRequest.chatId,
-            text = localizationText?.get(FINANCE_MANAGE_TEXT.name).toString(),
+            text = localizationText?.get(FINANCE_MANAGE_TEXT.name).textIsNotUploaded(),
```

Comment:
consider writing some wrapper in your textService where you'll handle all nullable errors instead of interacting with the map directly. because ton of textIsNotUploaded is a little different from a ton of toString

Reply:
Got it, I'll change that